templates/pricing/desktop.html

```diff
@@ -74,176 +71,326 @@ <h2 class="p-heading--4">Additional benefits for Enterprise Services Plus</h2>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">CIS Benchmark configuration and support</p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Snap Store configuration</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">
+                  Snap Store proxy deployment and local snap package repository.
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">CIFS/SMB configuration</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">
+                  Workstations configured to mount CIFS/SMB file systems from a third-party source
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row" rowspan="4">Security Configuration</th>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Installed software audit</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Installed software audit</p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Custom firewall enablement</p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Anti-virus/anti-malware solution support</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+            </tr>
+            <tr>
+              <td role="presentation"></td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Encrypted Volume Support</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Third party encryption support</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <div class="row">
+          <div class="col-start-large-4 col-6">
+            <div class="p-cta-block">
```

Comment:
Similar to the comment on the `p-cta-block` in the hero section, it adds additional padding to the bottom. Maybe we can just wrap the CTA in a `p` and add the `hr` manually?